backend/services/src/main/java/com/crimealert/services/UserService.java

```diff
@@ -206,7 +285,8 @@ private Document registerHelper(User user)
         EncodedPassword encoded = getPasswordUtil().generateUserPassword(user.getPassword());
         document.append(UserConstant.PASSWORD, encoded.getEncodedPassword());
         document.append(UserConstant.SALT, encoded.getSalt());
-        document.append(UserConstant.ENABLE_NOTIFICATIONS, user.getEnableNotifications());	
+        document.append(UserConstant.ENABLE_NOTIFICATIONS, false);	
+        document.append(UserConstant.VERIFICATION, false);	
```

Comment:
If a verified user tries to update password, will this set the user verification to false?